pkgs/applications/networking/ids/bro/default.nix

```diff
@@ -2,16 +2,20 @@
 , geoip, gperftools, python, swig }:
 
 stdenv.mkDerivation rec {
-  name = "bro-2.5.5";
+  name = "bro-2.6.2";
 
   src = fetchurl {
     url = "https://www.bro.org/downloads/${name}.tar.gz";
```

Comment:
With the renaming of the project, this address now forwards to the new domain. So perhaps the domain should already be updated.

```nix
url = "https://www.zeek.org/downloads/${name}.tar.gz";
```